lib/dai-plugin-mcd/src/CdpManager.js

```diff
@@ -34,6 +36,21 @@ export default class CdpManager extends LocalService {
     return new ManagedCdp(id, ilk, this);
   }
 
+  //todo: add in caching here??
+  async getDebtAllCdps(proxyAddress, descending = true) {
```

Comment:
Let's call this `getCombinedDebtValue` -- the `DebtValue` part would be consistent with `getDebtValue` in `ManagedCdp`, and the fact that this method is in `CdpManager` I think gives enough context that people should understand this to be an operation across all CDPs.